exporters/otlp/test/otlp_http_exporter_test.cc

```diff
@@ -106,7 +106,7 @@ class OtlpHttpExporterTestPeer : public ::testing::Test
   void ExportJsonIntegrationTest()
   {
     auto mock_otlp_client =
-        OtlpHttpExporterTestPeer::GetMockOtlpHttpClient(HttpRequestContentType::kJson);
+        OtlpHttpExporterTestPeer::GetMockOtlpHttpClient(HttpRequestContentType::kBinary);
```

Comment:
Can you check this change again, as the function name indicates this doing http/json test?

Reply:
pushed, thanks.

Reply:
I meant this function does the JSON test, it can remain unchanged.